plasmapy/simulation/particle_tracker.py

```diff
@@ -285,9 +285,31 @@ def save_now(self):
 
 
 class ParticleTracker:
-    """
-    General particle tracer.
+    r"""A many body particle tracker for electric and magnetic fields with no particle-particle interactions.
+
+    Particles are created and pushed through calculated E and B fields. These fields are specified as part of a grid
+    which can then be interpolated to determine local field strength. The time step used can be specified, otherwise an
+    adaptive time step is calculated based off grid resolution and the particle's gyroradius.
+
+    The simulation is terminated when a specified stop condition is met. The results of a simulation can be exported
```

Comment:
Stopping conditions are specified as instances of the ___ class as arguments to the `run` method. 

Save routines are specified as instances of the __ class as arguments to the `run` method.